scripts/trim-low-abund.py

```diff
@@ -100,19 +106,20 @@ def main():
     ###
 
     K = args.ksize
-    HT_SIZE = args.min_hashsize
-    N_HT = args.n_hashes
+    HT_SIZE = args.min_tablesize
```

Comment:
On Wed, Feb 11, 2015 at 08:07:50AM -0800, Michael R. Crusoe wrote:
> > @@ -100,19 +106,20 @@ def main():
> >      ###
> >  
> >      K = args.ksize
> > -    HT_SIZE = args.min_hashsize
> > -    N_HT = args.n_hashes
> > +    HT_SIZE = args.min_tablesize
> 
> The common usage is to just reference `args.whatever` directly.

Keeping K, as it is used elsewhere and it is quite clear :)